<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passwordChecker</class>
 <widget class="QMainWindow" name="passwordChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>253</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>638</width>
    <height>253</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>638</width>
    <height>253</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(226, 229, 255);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>620</width>
        <height>235</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>620</width>
        <height>235</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Password Checker</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputPassword">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(170, 85, 255);
	border-radius: 5px;
}

QLineEdit:focus{
	border: 2px solid rgb(134, 88, 234);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="checkBtn">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>150</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(170, 85, 255);
	border-radius: 5px;
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	background-color: rgb(134, 88, 234);
}</string>
       </property>
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
      <widget class="QLabel" name="responseText">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>611</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>password has been hacked 345 times, you should change your password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>441</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Check if your password has previously been exposed to data breaches. </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Uses pwnedPassword API (https://haveibeenpwned.com/)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
      <widget class="QLabel" name="statusText">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>60</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(0, 170, 127);
}</string>
       </property>
       <property name="text">
        <string>Connected</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
